bear-requirements.yaml

```diff
@@ -1,223 +0,0 @@
-# This is an automatically generated file.
```

Comment:
I don't there's any need to delete this file. Can you please explain why you did so?

Reply:
@shikharvaish28 Thats by mistake but I have re-edit it but if can you please me what you actually mean by change shortlog ,I have done that to PEP257 already

Reply:
Go through [this](https://api.coala.io/en/latest/Developers/Writing_Good_Commits.html) once @AkshJain99

Reply:
Okay i got it but I can't write commit shortlog more than 50 character,so how can I mention PEP8 to PEP 257??

Reply:
Well, that's up to you how you adjust it because the PR can't be merged without it 😉

Reply:
okay,  can you tell me if it is satisfactory now or not?

Reply:
Please add this file again.